src/lib/index.ts

```diff
@@ -215,10 +214,10 @@ export class GoogleToken {
     }
 
     const toSign = {
-      header: {alg: 'RS256', typ: 'JWT'},
+      header: {alg: 'RS256'} as jws.Header,
       payload,
       secret: this.key
-    };
+    } as jws.SignOptions;
```

Comment:
Do you still need this typecast even when you typecast `header`?

Reply:
No :)